### PR TITLE
Example of what the video guides might look like

### PR DESCRIPTION
**Pull request by mcclowes**
# Description

/introduction/first-ten-minutes

Thoughts?

### CODE REVIEW

File: docs/introduction/first-ten-minutes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Video: Your first ten minutes with Codat"
```
Comment (d-laing) [inline docs/introduction/first-ten-minutes.md:2]: Would be good to see how this wraps in the sidebar.

File: docs/introduction/first-ten-minutes.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+description: "From signing up to syncing in 10 minutes."
```
Comment (d-laing) [inline docs/introduction/first-ten-minutes.md:3]:
```suggestion
description: "From signing up to syncing data in 10 minutes."
```

File: docs/introduction/first-ten-minutes.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+---
+
+import LocalStorageChecklist from "@components/global/LocalStorageChecklist";
+
+## Your first ten minutes with Codat
+
+In this video we'll sign up to Codat, set up the Xero integration and pull data from Xero.
```
Comment (d-laing) [inline docs/introduction/first-ten-minutes.md:10]:
```suggestion
In this video we'll sign up to Codat, set up the Xero integration, and pull data from Xero.
```

**Review by d-laing — CHANGES_REQUESTED** [review]
Few comments on the text.

File: docs/introduction/first-ten-minutes.md (hunk 1 above)
Comment (mcclowes) [reply docs/introduction/first-ten-minutes.md:2]: It's definitely too long yeah

**Pull request merged**